samples/core/tfx-oss/README.md

```diff
@@ -20,8 +20,8 @@ then activate the environment.
 
 Install TFX and Kubeflow Pipelines SDK
 ```
-pip3 install tfx==0.13.0 --upgrade
-pip3 install kfp --upgrade
+pip3 install 'tfx==0.14.0' --upgrade
+pip3 install 'kfp>=0.1.31' --upgrade
```

Comment:
Should it be 'kfp>=0.1.31.2'?

Reply:
The compatibility in TFX's KubeflowDagRunne was only for test (which is fixed anyways https://github.com/tensorflow/tfx/commit/ea85bfef1a44e52df34202f374613c043be7a4d9#diff-642c05352d80030caf1d9d3fe41f9740), but there was no compatibility issue in orchestrator itse.f

Is there any other reason to prefer 0.1.31.2 or higher?

Reply:
`0.1.31.2` is just the current latest version.
Usually the latest version is the best, containing all the bug fixes.
The 0.1.32 will be released tomorrow which will have some more fixes.